### DISCUSSION TITLE
Disable clicking per-site

### DISCUSSION THREAD

**Discussion by ghost** [dhowe/AdNauseam#1831] [Ideas]
I suppose this is a bit silly due to what this extension is aiming to do, but I recall hearing that some Google accounts have been suspended for the use of this service. I use YouTube as a source of entertainment (logged in), and would like to partake in this cause: but I also wouldn't like to run the risk of my Google account being suspended. I'm using Firefox and am blocking unneeded cookies (using [Cookie Auto Delete](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete)), so hopefully Google doesn't do anything with my account due to the use of this extension on other sites. Of course they'll know it's me, but I'm thinking they wouldn't do anything because it's not directly tied to my account.
Is this possible? I looked through the settings to find anything that would make the extension to only work as a regular ad-blocker on set pages, but didn't see anything.

I also know that it's just a bad idea to use Google if you're trying to have any semblance of online privacy, but well. Here I am. I do avoid it where it's reasonable though.

**Comment by dhowe** [comment]
thanks @Hedgehog0091, this would, I think, be useful.

@cqx931 I believe we discussed this previously - do you happen to remember where/when?
